Chapter 6 Transcription

gregmacfarlane opened the pull request.

gregmacfarlane committed:
Add example data loading and log-likelihood

You can take this out, relocate it, or do whatever you think would be valuable.

gregmacfarlane committed:
Show how to estimate constants only and null models

SamKLau committed:
Added text for Chapter 5

SamKLau committed:
added formatting

gregmacfarlane committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

gregmacfarlane committed:
Add code on conducting tests

Commit:
Add Equation 5.13 to 5.18 and their symbols in the text

barnesmaxmax committed:
Add data input fix and equations 5.7-5.12

Added possible data input fix, the data needs to be checked for accuracy still. Input equations 5.7-5.12.

camlunt committed:
Equations 5.19-5.23

camlunt committed:
Fixed some formatting issues in equations 5.13-5.18

barnesmaxmax committed:
Add attempt at indexing SF data

Attempted to index the SF data and to create a choice model to see if indexing was done right.

barnesmaxmax committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

gregmacfarlane committed:
Cleanup data conversion, and show how to make tables for fig 5.1

gregmacfarlane committed:
Table cleanup

SamKLau committed:
Changes in text

SamKLau committed:
Equations 5-1 to 5-6

barnesmaxmax committed:
Add table formatting for figure 5.1

Table formatting added, along with alternate level labels added.

SamKLau committed:
worked on Figure 5.2

SamKLau committed:
math in code

SamKLau committed:
Merge branch 'transcribe5' of github.com:byu-transpolab/modechoice into transcribe5

SamKLau committed:
Table 5-3 and Figure 5-2

SamKLau committed:
added data folder and edits to table 5-3 and figure 5-2

camlunt committed:
Table 5-1 (mostly complete)

Still needs formatting for the percents as well as data for the second column

camlunt committed:
First attempt at Table 5-4

I can't get the coefficients table in summary(base_model) into a useable format (e.g., dataframe)

camlunt committed:
First attempt at Table 5-12 and Figure 5-7

Need understanding of time value of money equation

barnesmaxmax committed:
Add Figure 5.3

Add figure 5.3 with confidence intervals.

Commit:
add figure 5.5

Commit:
Table 5-10 incomplete

barnesmaxmax committed:
Add Table 5.8

Add Table 5.8 and add knitr, and kableextra packages to top of script.

loganbennett93 committed:
Equations 5.24 through 5.34

camlunt committed:
Rearranged text-figure order

Figure 5.5 was placed too early in the text; it was moved to be a paragraph later

loganbennett93 committed:
Add equations 5.24 through 5.34

barnesmaxmax committed:
Add start to Table 5-9

loganbennett93 committed:
Add Table 5-5

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

camlunt committed:
Added Figure 5.4

camlunt committed:
Fixed Typos in equation tags

Also committed the second half of the commit called "Rearranged text-figure order"

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

camlunt committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

barnesmaxmax committed:
Re-add figure 5.3

Figure 5.3 got removed during a conflict resolution. I've re-added it here.

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe5' into transcribe5

loganbennett93 committed:
Add Tables 5-6 and 5-13, and figures 5.6 and 5.8

camlunt committed:
Colors for Figure 5-4

camlunt committed:
Table 5-12 and Figure 5.7

barnesmaxmax committed:
Add table 5.9 and fix for table 5.2

gregmacfarlane committed:
Merge branch 'transcribe4' into transcribe5

loganbennett93 committed:
Add Table 5=7

This table has the correct coefficients and log likelihood values, but has incorrect rho-square values and is missing a number of other statistics given in the table from the original text. I'm not exactly sure how to recreate those statistics.

barnesmaxmax committed:
Edits for Table 5-9

Add equation edits to produce correct data for table 5-9.

gregmacfarlane committed:
Replace likelihood ratio test table with calculations

Note that you can pull the likelihood values directly from the model object.

gregmacfarlane committed:
Minor formatting

I think the basic kable_styling looks good, but the kable_classic has been used. I'll just do a find and replace at some point, so not a big deal. Also, use theme_bw() for the ggplot theme.

gregmacfarlane committed:
Replace constraint with removed variable

You have to have one generic variable in every formula. But instead of giving it a value and constraining it to be one, you can simply pass in a `1` to show that there are no additional variables.

gregmacfarlane committed:
Show how to add digits to the modelsummary output

the `fmt` argument is your friend. ?modelsummary gives a few different methods.

barnesmaxmax committed:
Add table fix for table 5-8

barnesmaxmax committed:
Add decimals to fig 5-8

barnesmaxmax committed:
Add Chapter 6 Text

Added text for Chapter 6.

barnesmaxmax commented:
Okay. Here's the breakdown of work for chapter 6. I've tried to break up the big and small tables between everyone as fairly as possible. Please use the SF data to make your tables. It makes it really simple, and it allows us to call in data from the tables elsewhere if needed. I think it would also be good to make sure we are naming chunks, equations, and everything else we are doing really well so that we can navigate this more easily because there is a lot going on in this chapter table wise. 

The tasks are as follows:
@barnesmaxmax 
- [ ] In text math
- [ ] Table 6.2
- [ ] Table 6.4
- [ ] Table 6.5
- [ ] Figure 6.1

@SamKLau 
- [ ] Bibliography References
- [ ] Table 6.6
- [ ] Table 6.7
- [ ] Eq. 6.1-6.5

@loganbennett93 
- [ ] Table 6.9
- [ ] Table 6.10
- [ ] Table 6.11
- [ ] In text section/chapter references (I think this is possible. Ask @gregmacfarlane for help if needed)

@camlunt 
- [ ] In text Table and Figure references (do this after everyone else makes their tables)
- [ ] Table 6.12
- [ ] Table 6.13
- [ ] Table 6.14
- [ ] Table 6.15

@bwang0601 
- [ ] Table 6.1
- [ ] Table 6.3
- [ ] Table 6.8
- [ ] Eq. 6.5-6.10

Commit:
merge

Merge branch 'transcribe5' of https://github.com/byu-transpolab/modechoice into transcribe5

# Conflicts:
#	05-mnl-estimation.Rmd

loganbennett93 committed:
Update Table 5-6 with Calculated LL test values

Commit:
Add table 5-11, error in thrid model, can't find why.

Commit:
Merge branch 'transcribe5' of https://github.com/byu-transpolab/modechoice into transcribe5

Commit:
Adjust Table 5-10

Commit:
Add eq 6.5 -6.10 and unlisted eq in between

camlunt committed:
Fix Table 5-4

gregmacfarlane committed:
Add chapter 6 setup chunk

gregmacfarlane committed:
Model segmentation example

loganbennett93 committed:
Update figure 5-6

loganbennett93 committed:
Begin Tables 6-9 and 6-10

I have the tables set up the way I want for the most part, but the values are a little off from what appears in the text. I also attempted to create in-text references, though I haven't figured it out completely yet.

camlunt committed:
Table 5-12

camlunt committed:
Table 5-1

camlunt committed:
code block for table 6-12

camlunt committed:
Functions for Value of Time (VOT)

camlunt committed:
First draft of Model 16W

camlunt committed:
Create new columns Travel Time

Includes TT for motorized modes only, TT for non-motorized modes only, and OVTT for motorized modes only.

camlunt committed:
Update on Model 16W

Adjusted the travel time variables

camlunt committed:
Update of Model 16W (still not quite right though)

camlunt committed:
No need for new columns

camlunt committed:
Model 17W and Code block for table 6-13

camlunt committed:
Code block for Table 6-14

camlunt committed:
Code block for Table 6-15

Commit:
add basic info for table 6.1, 6.3, and 6.8

barnesmaxmax committed:
Add in text math and base of table 6-5

barnesmaxmax committed:
Equation changes for table 6-5

loganbennett93 committed:
Add Table 6-11 and update Table 6-9

I've incorporated the pertinent needed for models 11w, 13w, 14w, 15w, and 16w, but even though they show up in the tables as in the text, the values are still slightly off from those in the text.

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

gregmacfarlane committed:
Just read in the data, don't need to remake it.

gregmacfarlane committed:
Commit modified worktrips file

gregmacfarlane committed:
Already read this file in above

Commit:
adjust tables

Commit:
Merge branch 'transcribe6' of https://github.com/byu-transpolab/modechoice into transcribe6

Commit:
update tables

Commit:
tables update

loganbennett93 committed:
Update Models 11w, 13w, 14w, 15w, 16w

These models appear in Table 6-9 and 6-11. I mutated the nm_ovtt column to divide by the distance. The values are closer to what appear in the text, but still slightly off.

Commit:
fix table 6-8

Commit:
Merge branch 'transcribe6' of https://github.com/byu-transpolab/modechoice into transcribe6

camlunt committed:
Update on VOT functions

loganbennett93 committed:
Update Model 11-w

We will need to change the constrained values in other models to equal the values given in the text

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

loganbennett93 committed:
Minor changes to models and remove constraints

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

Commit:
adjust table 6.1, 6.3, and 6.8. w/out setting shared ride 2 = 3+

bwang0601 committed:
merge

SamKLau committed:
Equations 6.1 to 6.3

SamKLau committed:
Equation committing

Merge branch 'transcribe6' of github.com:byu-transpolab/modechoice into transcribe6

# Conflicts:
#	06-specification.Rmd

SamKLau committed:
Bibliography References

SamKLau committed:
ch. 5 footnotes

SamKLau committed:
chapter 6 footnotes

SamKLau committed:
equations 6.1-6.4

SamKLau committed:
start of table 6-6 and 6-7

gregmacfarlane committed:
Edits to get book.bib to knit

gregmacfarlane committed:
Remove demo log-likelihood with fishing dataset

gregmacfarlane committed:
Minor table cleanup

gregmacfarlane committed:
Work on value-of-time tables

gregmacfarlane committed:
Show market share in percent

gregmacfarlane committed:
Replace references to table 5-2 with tags. Also various text cleanup

gregmacfarlane committed:
Update worktrips file

This is just a change that results when I run the code, but obviously the change hadn't been committed.

loganbennett93 committed:
Update Table 6-10

This table now gives the correct values of VoT with coefficients drawn straight from the models. However, the model coefficients are still a little off

camlunt committed:
Table 6-12

The first column is calculated from model coefficients. The second column uses hard-coded numbers. I can't figure out where the 0.47, 0.77, and 0.62 come from.

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

loganbennett93 committed:
Begin Table 6-4

barnesmaxmax committed:
Fix Table 6-5

Table 6-5 has correct values (or as correct as possible).

barnesmaxmax committed:
Change duplicated chunk name

camlunt committed:
Commented out later version of VOT equations

camlunt committed:
Model 17W

camlunt committed:
Segmentation by auto ownership

camlunt committed:
Segmentation by gender

Using femdum doesn't result in the correct parameters, but it's the best option I've come up with so far.

barnesmaxmax committed:
Update Table 5-8 Code

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

barnesmaxmax committed:
Merge branch 'transcribe5' into transcribe6

barnesmaxmax committed:
Add coefficient name fixes in table 6-1

gregmacfarlane committed:
Provide good example of chunk names and labels

loganbennett93 committed:
Auto stash before merge of "transcribe6" and "origin/transcribe6"

loganbennett93 committed:
Add Table 6-4

camlunt committed:
Added constraints and filters to Model 17W and the segmented models

camlunt committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

camlunt committed:
Fixed stashing issue for Table 6-4

loganbennett93 committed:
Merge remote-tracking branch 'origin/transcribe6' into transcribe6

loganbennett93 committed:
Fix Table 6-4

SamKLau committed:
Table 6-4

loganbennett93 committed:
Add in-text chapter/section references

I updated only those headings that were called out in Chapter 6, with a simple notation of chapters and sections followed by their respective numbers

barnesmaxmax committed:
Start Table 6-2

barnesmaxmax committed:
Add partial table 6-2

gregmacfarlane committed:
Add cross-references to section headers in chapter 5

gregmacfarlane committed:
Clean up setup chunk printing and comments

gregmacfarlane committed:
Fix multi-level markdown list

gregmacfarlane committed:
Label survey design sections

gregmacfarlane committed:
Correct table references and mild table structure edits for case / alt discussion

gregmacfarlane committed:
Basic cleanup

gregmacfarlane committed:
Simplify trip-alt table

gregmacfarlane committed:
Update numberline image link, knit book

gregmacfarlane committed:
Merge branch 'transcribe5' into transcribe6

camlunt committed:
Table 6-12

gregmacfarlane committed:
Build doc

gregmacfarlane committed:
Build table for model 7 values of time

gregmacfarlane committed:
Edit income interaction table

barnesmaxmax committed:
Add changes to table 6.2 and figure 6.1

Add changes in an attempt to get these items to show correct information.

barnesmaxmax committed:
Add changes to table 6-2 and table 6-1

SamKLau committed:
merged commits with previous branch

Merge branch 'transcribe6' of github.com:byu-transpolab/modechoice into transcribe6

# Conflicts:
#	06-specification.Rmd

SamKLau committed:
merge from previous requests

SamKLau committed:
updated equations

SamKLau committed:
adjustments to equations 6.4 and 6.5

SamKLau committed:
Tables 6.6 and Tables 6.7

SamKLau committed:
updates to equations 6.5 and 6.6, tables 6.6 and 6.7

camlunt committed:
Format changes to Table 6-12

camlunt committed:
Fixed Table 6-13

camlunt committed:
Added caption to Table 6-14

camlunt committed:
Fixed formatting in bullet points

camlunt committed:
Added caption for Table 6-15

gregmacfarlane committed:
Clean up references in first half of chapter 6

gregmacfarlane committed:
Delete unnecessary intermediate build files

gregmacfarlane committed:
Clean up multiline equations in chapter 6

gregmacfarlane committed:
Fix cacheing and other page options

gregmacfarlane committed:
Fix some equation labels

gregmacfarlane committed:
Fix ben akiva reference

gregmacfarlane committed:
Clean up code in vot ratio figure

gregmacfarlane committed:
Build site
